lib/webauthn/authenticator_assertion_response.rb

```diff
@@ -4,22 +4,37 @@
 
 module WebAuthn
   class AuthenticatorAssertionResponse < AuthenticatorResponse
-    def initialize(authenticator_data:, **options)
+    def initialize(authenticator_data:, signature:, **options)
       super(options)
 
       @authenticator_data_bytes = authenticator_data
+      @signature = signature
     end
 
-    def valid?(original_challenge, original_origin)
+    def valid?(original_challenge, original_origin, credential_public_key:)
       valid_type? &&
         valid_challenge?(original_challenge) &&
         valid_origin?(original_origin) &&
-        authenticator_data.user_present?
+        authenticator_data.user_present? &&
+        valid_signature?(public_key_bytes: credential_public_key)
     end
 
     private
 
-    attr_reader :authenticator_data_bytes
+    attr_reader :authenticator_data_bytes, :signature
+
+    def valid_signature?(public_key_bytes:)
```

Comment:
Why did you opted to use a hash arg format instead of a single regular param? Just curious :)

Reply:
No strong reason really.

I'll change it so it's kind of consistent with the rest of the `valid_*` methods.